extensions/wikia/WikiaInYourLang/modules/ext.wikiaInYourLang.js

```diff
@@ -112,7 +113,7 @@ require(
 			$('.global-notification.notify').click(function (event) {
 				if (event.target.parentElement.className.indexOf('close') !== -1) {
 					onNotificationClosed();
-				} else if (event.target.id.indexOf('wikia-in-your-lang-link') !== -1) {
+				} else if (event.target.className.indexOf('text') !== -1) {
```

Comment:
Minor one. As you're already using jQuery, it makes sense to use it here as well (to increase the readability of the code:

```js
var $target = $(event.target);

if ($target.hasClass('text') {
  // ..
}
```